pages/pages.go

```diff
@@ -186,11 +193,75 @@ func toHTML(filename string) (string, error) {
 	buf = blackfriday.MarkdownOptions(buf, renderer, blackfriday.Options{
 		Extensions: blackfriday.EXTENSION_AUTO_HEADER_IDS | blackfriday.EXTENSION_TABLES | blackfriday.EXTENSION_FENCED_CODE,
 	})
-	return string(buf), nil
+	body := html.Node{
+		Type:     html.ElementNode,
+		DataAtom: atom.Body,
+		Data:     "body",
+	}
+	nodes, err := html.ParseFragment(bytes.NewReader(buf), &body)
+	if err != nil {
+		return "", err
+	}
+	for _, node := range nodes {
+		body.AppendChild(node)
+	}
+	var removeSensitive func(n *html.Node) []*html.Node
+	removeSensitive = func(n *html.Node) []*html.Node {
+		removeDeep := false
+		removeShallow := false
```

Comment:
A bit unnice to have 2 flags since they are mutually exclusive, would have been nicer to have something like a three state enum (eg. `deep`, `shallow`, `none`). But that is maybe overkill in this instance, and notably not that clean looking in go (I think), so I think this is a fine solution.

Reply:
Yeah I agree, but I think the idiomatic thing is something like this:
```go
type RemoveMethod int
const (
    Keep RemoveMethod = iota
    RemoveDeep
    RemoveShallow
)
var removeMethod RemoveMethod
/* ... */
switch removeMethod {
case Keep:
    break
case RemoveDeep:
    return nil
case RemoveShallow:
    /* ... */
default:
    panic("Invalid remove method")
}
```
Which isn't that nice IMO

Reply:
yeah, exactly :/ not that nice in go